packages/turf-interpolate/test/types.ts

```diff
@@ -0,0 +1,16 @@
+import * as linearc from '../index'
```

Comment:
🤦‍♂️ Looks like this file came from the previous repo 🤣 

I've been putting `types.ts` at the root directory that way we don't forget it.

One day... all modules will have `types.ts` to test for the Typescript definitions.

So far this one looks like this:

```typescript
import * as interpolate from './';
import {point, featureCollection} from '@turf/helpers';

const cellSize = 1;
const property = 'pressure';
const weight = 0.5;
const units = 'miles';
const points = featureCollection([
  point([1, 2]),
  point([12, 13]),
  point([23, 22]),
]);

const grid = interpolate(points, cellSize, property, units, weight);
grid.features[0].properties.pressure;
```